src/components/TxFlow/TokenTransfer/TokenTransferFlow.tsx

```diff
@@ -0,0 +1,37 @@
+import { TokenTransferStepper } from '@/components/TxFlow/TokenTransfer/index'
+import { SendTxType } from '@/components/tx/modals/TokenTransferModal/SendAssetsForm'
+import CreateTokenTransfer from '@/components/TxFlow/TokenTransfer/CreateTokenTransfer'
+import ReviewTokenTransfer from '@/components/TxFlow/TokenTransfer/ReviewTokenTransfer'
+import TxLayout from '@/components/TxFlow/common/TxLayout'
+import { ProgressBar } from '@/components/common/ProgressBar'
+
+const initialData = {
+  recipient: '',
+  tokenAddress: '',
+  amount: '',
+  type: SendTxType.multiSig,
+}
+
+const TokenTransferFlow = ({ txNonce }: { txNonce?: number }) => {
```

Comment:
The naming is a bit inconsistent, this is the only transaction type named with "Flow" at the end.

Reply:
We were also thinking of naming `RejectTx` to `RejectTxFlow` but it only has one step and we didn't use the Stepper there so we left it as `RejectTx`.